extensions/cms/package.nls.json

```diff
@@ -0,0 +1,107 @@
+{
+	"cms.displayName": "CMS",
+	"cms.description": "Central Management Servers",
+	"cms.title": "Central Management Servers",
+	"cms.resource.explorer.title": "Central Management Servers",
+	"cms.resource.refreshServerGroup.title": "Refresh Server Group",
+	"cms.resource.refreshCmsServer.title": "Refresh Central Management Server",
+	"cms.resource.deleteRegisteredServer.title": "Delete Registered Server",
+	"cms.resource.addRegisteredServer.title": "Add Registered Server",
+	"cms.resource.deleteServerGroup.title": "Delete Server Group",
+	"cms.resource.addServerGroup.title": "Add Server Group",
+	"cms.resource.registerCMSServer.title": "Add Central Management Server",
+	"cms.resource.deleteCMSServer.title": "Delete Central Management Server",
+
+	"mssql.configuration.title": "MSSQL configuration",
```

Comment:
why are these `mssql` options in the CMS extension?

Reply:
These connection options are from `mssql`. I can change the tag to `cms`.